LayoutTests/http/wpt/dom-ranges-live-range/Range-mutations-deleteData.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html><!-- webkit-test-runner [ LiveRangeSelectionEnabled=true ] -->
```

Comment:
oh, I think we want to delete this entire directory. This is a copy of `LayoutTests/imported/w3c/web-platform-tests/selection`.